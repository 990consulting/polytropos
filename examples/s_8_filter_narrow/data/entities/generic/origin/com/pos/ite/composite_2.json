{
  "1": {"temporal_int": 42},
  "2": {"temporal_int": 100},
  "3": {"temporal_int": 97},
  "immutable": {
    "immutable_int": 19
  }
}
