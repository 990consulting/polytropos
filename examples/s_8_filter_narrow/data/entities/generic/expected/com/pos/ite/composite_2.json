{
  "3": {"temporal_int": 97},
  "immutable": {
    "immutable_int": 19
  }
}
